scripts/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
```

Comment:
It is unclear to me why `/scripts` requires a `Project.toml`? What is the motivation?